### PR TITLE
test add 2 files (bot test)

### PR DESCRIPTION
**Pull request by drok-feedback**
fasdfasdfa

Backport 9dc3f61c is needed to enable the blah1 feature
Backports b4d189ac..9dc3f61c are needed for performance for blah 2 feature
More edit after drok approved.

### CODE REVIEW

File: file5 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+File6
+vzxc
+vz
```
Comment (drok) [inline file5]: file6 line 3

Comment (drok) [inline file5]: comment on new lines 2 and 3 [above comment](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936051891) [this comment](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936052643)
[first](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936071670)

todo:
 - [ ] Action A
 - [ ] Action B

questions:
 - Is it A? [ ] B? [ ] C? [ ]

**Review by drok — COMMENTED** [review]
some general comments and includes specific comments on file 5

File: file 6 [hunk 2]
```diff
@@ -0,0 +1,9 @@
+fasdfad
+fa
+sdf
+ads
```
Comment (drok) [inline file 6:4]: hello lines 3-7
 - [1](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936052643)
 - [2](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936051891)

File: file5 (hunk 1 above)
Comment (drok) [reply file5]: how to quote
```suggestion
zzz
vzxc
vzfg
```

> comment on new lines 2 and 3 [above comment](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936051891) [this comment](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936052643) [first](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936071670)
> 
> todo:
> 
> * [ ]  Action A
> * [ ]  Action B
> 
> questions:
> 
> * Is it A? [ ] B? [ ] C? [ ]

File: file 6 (hunk 2 above)
Comment (drok) [reply file 6:4]: > hello lines 3-7
> 
> * [1](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936052643)
> * [2](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936051891)

Is commit 9dc3f61 a bugfix or new feature?
 - [ ] bugfix
 - [ ] new feature

Is it related to #3 ?
 - [ ] yes
 - [ ] no

Comment (drok-feedback) [reply file 6:4]: > > hello lines 3-7
> > 
> > * [1](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936052643)
> > * [2](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936051891)
> 
> Is commit [9dc3f61](https://github.com/drok/octokit.net/commit/9dc3f61c69920a3f33f67ad1188c1c3c6e9b573c) a bugfix or new feature?
> 
> * [ ]  bugfix
> * [ ]  new feature
> 
> Is it related to #3 ?
> 
> * [ ]  yes
> * [ ]  no

Comment (drok-feedback) [reply file 6:4]: > hello lines 3-7
> 
> * [1](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936052643)
> * [2](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936051891)

New comment without being collaborator:
Is it related to Is commit 9dc3f61 a bugfix or new feature?
 - [x] bugfix
 - [ ] new feature

It is related to #3

**Review by drok — APPROVED** [review]
looks good to me

**Review by drok — APPROVED** [review]
this is bots' review?

**Comment by drok-feedback** [comment]
> * [x]  Explain the draft merges
> * [x]  Explain the bundled topics
  > * [x]  Document the backports as **errata** in the pull request. They should have been documented before, but later is better than never.

**Comment by drok** [comment]
# Merge summary (tags & guides included, other refs omitted, uninteresting HEAD commits omitted):
## Backports (1)
  * 5910297 pre E3 merge
## Merge Graph
```mermaid
%%{init: { 'theme': 'neutral', 'gitGraph': {'rotateCommitLabel': true},'themeVariables': { 'git3': '#990000' }} }%%
gitGraph
 branch feat-2 order: 3
 checkout main
 commit id: " "
 branch your-branch order: 2
 checkout main
 commit id: "3e86c96"
 branch trivial-backport-2 order: 1
 branch trivial-backport-1 order: 1
 commit id: "3e86c96..5910297"
 checkout trivial-backport-2
 commit id: "bp2"
 checkout your-branch
 commit id: "3e86c96..039abf7"
 commit id: "5910297"
 merge trivial-backport-1
 merge trivial-backport-2
 checkout main
 merge your-branch tag: "This PR"
```

**Comment by drok** [comment]
# Merge summary (tags & guides included, other refs omitted, uninteresting HEAD commits omitted):
## Backports (1)
  * 5910297 pre E3 merge
## Merge Graph
```mermaid
%%{init: { 'theme': 'neutral', 'gitGraph': {'rotateCommitLabel': true},'themeVariables': { 'git2': '#990000', 'gitBranchLabel2': '#ffffff' }} }%%
gitGraph
 commit id: " "
 branch your-branch order: 3
 checkout main
 commit id: "3e86c96"
 branch trivial-backport-1 order: 1
 checkout trivial-backport-1
 commit id: "3e86c96..5910297"
 checkout your-branch
 commit id: "3e86c96..039abf7"
 commit id: "5910297"
 merge trivial-backport-1
 checkout main
 merge your-branch tag: "This PR"
```

**Comment by drok** [comment]
# Proposed merge summary
## Drafts (3)
  * c358fc4 Merge branches 'sh/hwblk' and 'sh/platform-updates'
  * 6e1922 Merge git://git.kernel.org/pub/scm/linux/kernel/gi...
  * a3bedd Merge branch 'sh/kfr2r09'
## Merge Graph
```mermaid
%%{init: { 'theme': 'neutral', 'gitGraph': {'rotateCommitLabel': true},'themeVariables': { 'git4': '#990000', 'gitBranchLabel4': '#ffffff' }} }%%
gitGraph
 commit id: " " tag: "v2.6.31-rc2"
 branch your-branch order: 3
 checkout main
 commit id: "..f686d8c1"
 checkout your-branch
 commit id: "69ca06 (V1)"
 checkout main
 merge your-branch
 checkout your-branch
 branch draft-1 order: 2
 branch draft-2 order: 2
 checkout draft-2
 commit id: "ae1cef6..c358fc4"
 checkout main
 commit id: "..c358fc4"
 merge draft-2
 checkout your-branch
 checkout draft-1
 commit id: "241ad11..6e1922"
 checkout main
 commit id: "6e1922"
 merge draft-1
 checkout your-branch
 checkout main
 commit id: "..fc6cca"
 checkout your-branch
 commit id: "..aea1f7 (V2)"
 checkout main
 merge your-branch
 checkout your-branch
 branch draft-3 order: 2
 branch subtopic-1 order: 4
 checkout subtopic-1
 commit id: "658874..2e6713"
 checkout your-branch
 commit id: "..2e6713"
 merge subtopic-1
 checkout draft-3
 commit id: "3730793..a3bedd"
 checkout main
 commit id: "..a3bedd"
 merge draft-3
 checkout your-branch
 commit id: "2e6713..2e6713 (V3)"
 checkout main
 commit id: "..133b170f"
 merge your-branch tag: "This PR"
 checkout your-branch
```

**Comment by drok** [comment]
# Proposed merge summary
## Drafts (3)
  * c358fc4 Merge branches 'sh/hwblk' and 'sh/platform-updates'
  * 6e1922 Merge git://git.kernel.org/pub/scm/linux/kernel/gi...
  * a3bedd Merge branch 'sh/kfr2r09'
## Merge Graph
```mermaid
%%{init: { 'theme': 'neutral', 'gitGraph': {'rotateCommitLabel': true},'themeVariables': { 'git4': '#990000', 'gitBranchLabel4': '#ffffff' }} }%%
gitGraph
 commit id: " " tag: "v2.6.31-rc2"
 branch your-branch order: 3
 checkout main
 commit id: "..f686d8c1"
 checkout your-branch
 commit id: "69ca06 (V1)"
 checkout main
 merge your-branch
 checkout your-branch
 commit id: "..4b0a84"
 branch draft-1 order: 2
 commit id: "..04e448 (V2d1)"
 checkout your-branch
 commit id: "..78af08"
 branch draft-2 order: 2
 commit id: "..2094e5? (V2d2)"
 checkout your-branch
 checkout main
 commit id: "..c358fc4"
 merge draft-2
 checkout your-branch
 checkout main
 commit id: "6e1922"
 merge draft-1
 checkout your-branch
 checkout main
 commit id: "..fc6cca"
 checkout your-branch
 commit id: "..aea1f7 (V2)"
 checkout main
 merge your-branch
 checkout your-branch
 branch draft-3 order: 2
 commit id: "..5bdef86 (V3d1)"
 checkout your-branch
 branch subtopic-1 order: 4
 checkout main
 commit id: "..a3bedd"
 merge draft-3
 checkout your-branch
 commit id: "..2e6713 (V3)"
 checkout main
 commit id: "..133b170f"
 merge your-branch tag: "This PR"
 checkout your-branch
```
@ git db547c31d1a65759d4be6b4bb3bf874c62ef4ef5

**Comment by drok** [comment]
# Proposed merge summary
## Drafts (3)
  * c358fc4 Merge branches 'sh/hwblk' and 'sh/platform-updates'
  * 6e1922 Merge git://git.kernel.org/pub/scm/linux/kernel/gi...
  * a3bedd Merge branch 'sh/kfr2r09'
## Merge Graph
```mermaid
%%{init: { 'theme': 'neutral', 'gitGraph': {'rotateCommitLabel': true},'themeVariables': { 'git4': '#990000', 'gitBranchLabel4': '#ffffff' }} }%%
gitGraph
 commit id: " " tag: "v2.6.31-rc2"
 branch your-branch order: 3
 checkout main
 commit id: "..f686d8c1"
 checkout your-branch
 commit id: "69ca06 (V1)"
 checkout main
 merge your-branch
 checkout your-branch
 commit id: "..4b0a84"
 branch your-branch-V2-draft-1 order: 2
 commit id: "..04e448 (V2d1)"
 checkout your-branch
 commit id: "..78af08"
 branch your-branch-V2-draft-2 order: 2
 commit id: "..2094e5? (V2d2)"
 checkout your-branch
 checkout main
 commit id: "..c358fc4"
 merge your-branch-V2-draft-2
 checkout your-branch
 checkout main
 commit id: "6e1922"
 merge your-branch-V2-draft-1
 checkout your-branch
 checkout main
 commit id: "..fc6cca"
 checkout your-branch
 commit id: "..aea1f7 (V2)"
 checkout main
 merge your-branch
 checkout your-branch
 branch your-branch-V3-draft-1 order: 2
 commit id: "..5bdef86 (V3d1)"
 checkout your-branch
 branch subtopic-1 order: 4
 checkout main
 commit id: "..a3bedd"
 merge your-branch-V3-draft-1
 checkout your-branch
 commit id: "..2e6713 (V3)"
 checkout main
 commit id: "..133b170f"
 merge your-branch tag: "This PR"
 checkout your-branch
```

@ git 40f2dcf3484c9592b0ce7394c5654dd372f129bb

**Comment by drok** [comment]
# Proposed merge summary
## Drafts (3)
  * c358fc4 Merge branches 'sh/hwblk' and 'sh/platform-updates'
  * 6e1922 Merge git://git.kernel.org/pub/scm/linux/kernel/gi...
  * a3bedd Merge branch 'sh/kfr2r09'
## Merge Graph
```mermaid
%%{init: { 'theme': 'neutral', 'gitGraph': {'rotateCommitLabel': true},'themeVariables': { 'git4': '#990000', 'gitBranchLabel4': '#ffffff' }} }%%
gitGraph
 commit id: " " tag: "v2.6.31-rc2"
 branch your-branch order: 3
 checkout main
 commit id: "..f686d8c1 zz"
 checkout your-branch
 commit id: "69ca06 (V1)"
 checkout main
 merge your-branch
 checkout your-branch
 commit id: "..4b0a84"
 branch your-branch-V2-draft-2 order: 2
 commit id: "..04e448 (V2d2)"
 checkout your-branch
 commit id: "..78af08"
 branch your-branch-V2-draft-1 order: 2
 commit id: "..2094e5? (V2d1)"
 checkout your-branch
 commit id: "..aea1f7"
 branch your-branch-V3-draft-1 order: 2
 commit id: "..5bdef86 (V3d1)"
 checkout your-branch
 checkout main
 commit id: "..fc6cca zz"
 checkout your-branch
 checkout main
 merge your-branch
 checkout your-branch
 checkout main
 commit id: "..c358fc4"
 merge your-branch-V2-draft-1
 checkout your-branch
 checkout main
 commit id: "6e1922"
 merge your-branch-V2-draft-2
 checkout your-branch
 branch subtopic-1 order: 4
 checkout subtopic-1
 commit id: "658874..2e6713"
 checkout your-branch
 commit id: "..2e6713"
 merge subtopic-1
 checkout main
 commit id: "..a3bedd"
 merge your-branch-V3-draft-1
 checkout your-branch
 checkout main
 commit id: "..133b170f"
 merge your-branch tag: "This PR"
 checkout your-branch
```

**Comment by drok** [comment]
# Proposed merge summary
## Drafts (3)
  * c358fc4 Merge branches 'sh/hwblk' and 'sh/platform-updates'
  * 6e1922 Merge git://git.kernel.org/pub/scm/linux/kernel/gi...
  * a3bedd Merge branch 'sh/kfr2r09'
## Merge Graph
```mermaid
%%{init: { 'theme': 'neutral', 'gitGraph': {'rotateCommitLabel': true},'themeVariables': { 'git4': '#990000', 'gitBranchLabel4': '#ffffff' }} }%%
gitGraph
 commit id: " " tag: "v2.6.31-rc2"
 branch your-branch order: 3
 checkout main
 commit id: "..f686d8c1"
 checkout your-branch
 commit id: "69ca06 (V1)"
 checkout main
 merge your-branch
 checkout your-branch
 commit id: "..4b0a84"
 branch your-branch/V2/draft-2 order: 2
 commit id: "..04e448 (V2d2)"
 checkout your-branch
 commit id: "..78af08"
 branch your-branch/V2/draft-1 order: 2
 commit id: "..2094e5? (V2d1)"
 checkout your-branch
 checkout main
 commit id: "..c358fc4"
 merge your-branch/V2/draft-1
 checkout your-branch
 checkout main
 commit id: "6e1922"
 merge your-branch/V2/draft-2
 checkout your-branch
 checkout main
 commit id: "..fc6cca"
 checkout your-branch
 commit id: "..aea1f7 (V2)"
 checkout main
 merge your-branch
 checkout your-branch
 branch your-branch/V3/draft-1 order: 2
 commit id: "..5bdef86 (V3d1)"
 checkout your-branch
 checkout main
 commit id: "..a3bedd"
 merge your-branch/V3/draft-1
 checkout your-branch
 commit id: "..2e6713 (V3)"
 checkout main
 commit id: "..133b170f"
 merge your-branch tag: "This PR"
 checkout your-branch
```
@ git d930a19b3d829aaa26c6720092599b14dd31724b

**Pull request closed without merging**